source/components/GlobalFooter/components/Svg.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Svg = ({ name, className, ...rest }) => (
+    <svg className={className} {...rest}>
+        <use xlinkHref={`#${name}`} />
```

Comment:
This will not work unless the images have been added as a sprite - check http://localhost:6060/#!/GlobalFooter

You should use assets directly like so https://wikia.github.io/react-common/#logowikia :
```
import LogoWikia from '@wikia/react-common/source/assets/LogoWikia';

<LogoWikia height="80" />
```

Reply:
Demo page itself works because on the demo page ALL THE ASSETS are already included on the page.

See also https://github.com/Wikia/react-common/blob/master/source/components/GlobalNavigation/components/CommunityBar/CommunityBar.js#L16